Superlista/Models/BridgeModels/UserModelConverter.swift

```diff
@@ -36,30 +36,38 @@ class UserModelConverter {
     }
     
     /**
-    This method converts our cloud CKUserModel structure to a local UserModel structure
+     This method converts our cloud CKUserModel structure to a local UserModel structure
      
      - Parameters:
-        - user: the cloud user to be converted - CKUserModel
+     - user: the cloud user to be converted - CKUserModel
      - Returns: the local version of the given cloud user - UserModel
      */
-    #warning("reavaliar funcao. Ela esta chamando outras recursivamente e portanto nunca sai daqui")
+#warning("reavaliar funcao. Ela esta chamando outras recursivamente e portanto nunca sai daqui")
     func convertCloudUserToLocal(withUser user: CKUserModel) -> UserModel {
         let id: String
         let name: String
         let customProducts: [ProductModel]
         var myLists: [ListModel] = []
         var sharedWithMe: [ListModel] = []
-                
+        
         id = user.id.recordName
-        name = user.name!
+        name = user.name ?? ""
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()